test/ownership-transfers.test.js

```diff
@@ -12,13 +13,19 @@ let accounts = require('./ae/accounts')
 let util = require('./util/util')
 let db = require('./util/db')
 let globalSetup = require('./global-setup')
+let amqpUtil = require('./util/amqp-util')
 
 describe('Ownership transfer tests', function() {
 
     before(async () => {
         await db.clearTransactions(adminWalletTx.hash)
     })
 
+    beforeEach( async () => {
+        await amqpUtil.init()
+        amqpUtil.clearAllMessages()
+    })
```

Comment:
There's no need to init queue before each of the tests in this test file. Also, tests are failing on CircleCI because of zombie process surviving after all the tests performed - so all the tests are OK but a NodeJS process stays alive . My hunch tells me it could be due to calling `amqpUtil.init()` in `beforeEach()` - in total 2 times, but `amqpUtil.stop()` is being called only once since it's called inside the `after()` block.

Either:

- call `amqpUtil.stop()` inside the `afterEach()` block

or

- call `amqpUtil.init()` inside the `before()` block 
- call `amqpUtil.clearAllMessages()` inside the `beforeEach()` block (you still want to reset all the messages before every single test)
- call `amqpUtil.stop()` inside the `after()` block

Latter approach looks better to me :) 

Please try this and see if it helps with the tests. To check this, simply run `npm test` on your machine and wait after all the tests have passed to verify that the process has died.